From 1319af9010d78d9c320fcd5eeb1c8a5da5a8c485 Mon Sep 17 00:00:00 2001
From: xuhuicong <xhc@rock-chips.com>
Date: Mon, 17 Apr 2017 15:43:59 +0800
Subject: [PATCH] drm/rockchip: fix hdmi no display when resume or switch
 resolution

spelling mistake cause hdmi phy configure error, now correct it

Change-Id: I548d76dd44e8d39e35b95138ec3d25b358cf3376
Signed-off-by: xuhuicong <xhc@rock-chips.com>
---
 drivers/gpu/drm/rockchip/dw_hdmi-rockchip.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rockchip/dw_hdmi-rockchip.c b/drivers/gpu/drm/rockchip/dw_hdmi-rockchip.c
index 65cc55a86478..81af1747566b 100644
--- a/drivers/gpu/drm/rockchip/dw_hdmi-rockchip.c
+++ b/drivers/gpu/drm/rockchip/dw_hdmi-rockchip.c
@@ -244,7 +244,7 @@ static int rockchip_hdmi_parse_dt(struct rockchip_hdmi *hdmi)
 			return -ENOMEM;
 		}
 		phy_table_size = val / 16;
-		of_property_read_u32_array(np, "rockchip,phy_table",
+		of_property_read_u32_array(np, "rockchip,phy-table",
 					   phy_config, val / sizeof(u32));
 		ret = rockchip_hdmi_update_phy_table(hdmi, phy_config,
 						     phy_table_size);
-- 
2.35.3

